src/Hydrator/HydratorInterface.php

```diff
@@ -9,11 +9,11 @@
 
 namespace Zend\Stdlib\Hydrator;
 
-use Zend\Stdlib\Extractor\ExtractionInterface;
+use Zend\Hydrator\HydratorInterface as BaseHydratorInterface;
 
 /**
  * @deprecated Use Zend\Hydrator\HydratorInterface from zendframework/zend-hydrator instead.
  */
-interface HydratorInterface extends HydrationInterface, ExtractionInterface
+interface HydratorInterface extends BaseHydratorInterface
```

Comment:
Thanks for catching that!  In the current state any methods typehinting on the deprecated `Zend\Stdlib\Hydrator\HydrationInterface` or `Zend\Stdlib\Extractor\ExtractionInterface` will no longer accept objects implementing `Zend\Stdlib\Hydrator\HydratorInterface`